packages/integration-tests/tests/api/transactions.test.ts

```diff
@@ -381,7 +462,20 @@ describe("Transactions", () => {
         );
```

Comment:
Probably some fields missed?
`
"items": [
        {
            "from": "0x3c0FD3279c2b714141aa731519b68Bd428454073",
            "to": "0x0000000000000000000000000000000000008001",
            "blockNumber": 16511607,
            "transactionHash": "0xbe1334c29408d1fe90a810fbda46da059f18cef033bfb08d75050c531b5c789e",
            "timestamp": "2024-02-06T10:44:38.831Z",
            "amount": "1503311400000000",
            "tokenAddress": "0x000000000000000000000000000000000000800A",
            "type": "fee",
            "tokenType": "ETH",
            "fields": null,
            "isInternal": false,
            "token": {
                "l2Address": "0x000000000000000000000000000000000000800A",
                "l1Address": "0x0000000000000000000000000000000000000000",
                "symbol": "ETH",
                "name": "Ether",
                "decimals": 18,
                "usdPrice": 2294.85,
                "liquidity": 275206159445,
                "iconURL": "https://assets.coingecko.com/coins/images/279/large/ethereum.png?1696501628"
            }
        },
`
https://[block-explorer-api.testnets.zksync.dev/transactions/0xbe1334c29408d1fe90a810fbda46da059f18cef033bfb08d75050c531b5c789e/transfers](https://block-explorer-api.testnets.zksync.dev/transactions/0xbe1334c29408d1fe90a810fbda46da059f18cef033bfb08d75050c531b5c789e/transfers)